lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2098,6 +2098,47 @@ def _process_marketplace_image_plan(
 
         return plan
 
+    def _generate_sku_capability(
+        self,
+        vm_size: str,
+        location: str,
+        cap_file: str,
+    ) -> AzureCapability:
+        # some vm size cannot be queried from API, and the capability
+        # may be queried from capability files through hooks.
+        capability_dict = plugin_manager.hook.azure_add_sku_capability(
```

Comment:
Is this supposed to match the hookspec `azure_update_sku_capability`? Or is this a different hook?

Reply:
I guess they should be the same.